website/src/layouts/base/header/MegaMenu.tsx

```diff
@@ -5,7 +5,9 @@ import type { WithClassName } from '../../../types/WithClassName.ts';
 export function MegaMenu({ className, children }: PropsWithChildren<WithClassName>) {
     return (
         <div className={`absolute left-0 w-screen bg-white shadow-lg ${className}`}>
-            <ul className='grid grid-cols-2 gap-4 border-2 border-gray-100 p-8 xl:grid-cols-5'>{children}</ul>
+            <ul className='grid max-h-[80vh] grid-cols-2 gap-4 overflow-y-scroll border-2 border-gray-100 p-8 lg:grid-cols-3 xl:grid-cols-5'>
```

Comment:
I just found this: https://developer.mozilla.org/en-US/docs/Web/CSS/overscroll-behavior
Seems like a good idea to apply here
```suggestion
            <ul className='overscroll-none grid max-h-[80vh] grid-cols-2 gap-4 overflow-y-scroll border-2 border-gray-100 p-8 lg:grid-cols-3 xl:grid-cols-5'>
```

Reply:
Good idea.

Reply:
(Actually it would be good to apply that to the `overlayThatClosesMenuOnClickOutside`, too, somehow, but I did not find a way to quickly do that. Let's keep it in mind for later. Doesn't matter much, but it would be nice if scrolling on the overlay did not scroll the page in the background)